.golangci.yml

```diff
@@ -96,6 +96,7 @@ linters:
     # we don't want to change code to Go 1.22+ yet
     - intrange
     - copyloopvar
+    - tenv
```

Comment:
```
WARN The linter 'tenv' is deprecated (since v1.64.0) due to: Duplicate feature another linter. Replaced by usetesting. 
```

Reply:
Should we enable `usetesting` instead? No idea atm, what it exactly does.